schema.json

```diff
@@ -1,2498 +1,2626 @@
-[

```

Comment:
@icatalina - is there a way to get the changes in here without causing the entire schema to differ?